charmhelpers/contrib/charmsupport/nrpe.py

```diff
@@ -484,3 +484,14 @@ def add_haproxy_checks(nrpe, unit_name):
         shortname='haproxy_queue',
         description='Check HAProxy queue depth {%s}' % unit_name,
         check_cmd='check_haproxy_queue_depth.sh')
+
+
+def remove_deprecated_check(nrpe, deprecated_services):
+    """
+    Remove checks fro deprecated services in list
+
+    :param NRPE nrpe: NRPE object to remove check from
+    """
```

Comment:
Please could we add `:type nrpe: ...` and param and type for `deprecated_services`?  Although it's patchy in the library, it's useful if we can fully spec the docstrings for new functions and when old ones are modified.

Reply:
Thank you for the comment, I think is OK now.